templates/page/details.html

```diff
@@ -20,9 +20,16 @@
 {% endblock breadcrumb %}
 
 {% block content %}
-  {% if not is_visible %}
+  {% if not page.is_visible %}
     <div class="alert alert-warning" role="alert">
-      {% blocktrans trimmed with date=page.available_on|date context "Page details text" %}
+      {% blocktrans context "Unavailable page detail text" %}
+        <strong>Warning!</strong>
+        You are previewing a page that is not visible.
+      {% endblocktrans %}
+    </div>
+  {% elif not is_visible %}
```

Comment:
Shouldn't it be `elif not page.is_visible`? @akjanik

Reply:
It is correct, even if a bit misleading. `page.is_visible` is object attribute, while `is_visible` is variable from the view:
https://github.com/mirumee/saleor/blob/b138c99e05ecc64983e2705992ee04680dc07196/saleor/page/views.py#L13
The point is to distinguish when we have page set as `publish` (not yet) and when `publish` toggle is unticked even if availability date has been chosen.